cocos/gfx/webgl/webgl-swapchain.ts

```diff
@@ -191,7 +191,7 @@ export function getExtensions (gl: WebGLRenderingContext): IWebGLExtensions {
 
     return res;
 }
-
+const premultipliedAlpha = false;
```

Comment:
Why need this variable?

Reply:
This attribute is used to obtain the format of the current swapchain. If it is false, the actual drawingBufferFormat is RGB8, which can lead to format mismatch errors when offscreen MSAA is blitted to the screen.

Reply:
In fact, the RGB8 format for the swapchain can bring some performance improvements to the rendering process of readPixels or renderTexture, but to avoid breaking compatibility, I still use RGBA8 when assigning the on-screen format.

Reply:
But it is const, it seems meaningless.